### PR TITLE
LPS-185122 Filter by tags & extensions modals display items from sites/asset libraries the user does not have permissions to

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
https://issues.liferay.com/browse/LPS-185122

How to test

Enable the feature flag "LPS-84424"

tags:

1. Create on site Global : globalTag1 , globalTag2
2. Create on site Liferay : tag1 , tag2
3. Add a new site (site2)
4. Create on site2 : tagSite2_1 , tagSite2_2
5. Go to Liferay Site
6. Go to Documents and Media
7. Go to Filter and Order > Tags

- Only   globalTag1 , globalTag2, tag1 and tag2 is shown

extensions:

1. Create on site Global new Document : Document.css
2. Create on site Liferay : document.png
3. Add a new site (site2) (if not created)
4. Create on site2 new Document : Document.doc
5. Go to Liferay Site
6. Go to Documents and Media
7. Go to Filter and Order > Extension

- only png extension is shown

- LPS-185122 show only the tags for the current group and Ancestors (Global)
- LPS-185122 add selectedGroupId to FileExtensionItemSelectorCriterion to filter the groups
- LPS-185122 baseline
- LPS-185122 add the new field of groups to the search content
- LPS-185122 add the current scoped group to the fileExtensionItemSelectorCriterion to filter only the extension for the group that I'm currently. On this case we only will see the scoped group not the ancestors because we only show Documents of the groups and for the local repositories.

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminManagementToolbarDisplayContext.java [hunk 1]
```diff
@@ -833,6 +833,13 @@ private String _getExtensionsItemSelectorURL() {
 		fileExtensionItemSelectorCriterion.setDesiredItemSelectorReturnTypes(
 			Collections.singletonList(new UUIDItemSelectorReturnType()));
 
+		ThemeDisplay themeDisplay =
+			(ThemeDisplay)_liferayPortletRequest.getAttribute(
+				WebKeys.THEME_DISPLAY);
+
+		fileExtensionItemSelectorCriterion.setSelectedGroupIds(
+			new long[] {themeDisplay.getScopeGroupId()});
```
Comment (marcogalluzzi) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminManagementToolbarDisplayContext.java:841]: The existing private attribute `_themeDisplay` can be used instead.

**Review by marcogalluzzi — CHANGES_REQUESTED** [review]
Very good! Just a very little change...

**Comment by AliciaGarciaGarcia** [comment]
@marcogalluzzi I pushed 2 commits with changes suggested and not.

thanks!

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by marcogalluzzi** [comment]
ci:test:stable

**Comment by marcogalluzzi** [comment]
Hi @AliciaGarciaGarcia please review this failing [test](https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/549835/testReport/com.liferay.content.dashboard.document.library.internal.item.selector.test/ContentDashboardFileExtensionItemSelectorViewTest/testGetData/) which seems related to your changes.

**Comment by AliciaGarciaGarcia** [comment]
added a fix for the 
[com.liferay.content.dashboard.document.library.internal.item.selector.test.ContentDashboardFileExtensionItemSelectorViewTest](https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/549835/testReport/com.liferay.content.dashboard.document.library.internal.item.selector.test/ContentDashboardFileExtensionItemSelectorViewTest) - [Failed 0 of last 25](https://testray.liferay.com/home/-/testray/case_results/2316133649/history)

and rebased with master

**Comment by marcogalluzzi** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:stop

**Comment by AliciaGarciaGarcia** [comment]
@marcogalluzzi I added one test with a new behavior
https://github.com/liferay-lima/liferay-portal/pull/4336/commits/02f10ca7d09f59fab4aa5def4f1fcc3a1f242770

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
sorry https://github.com/liferay-lima/liferay-portal/pull/4336/commits/27551626155fc00a27a998d55b7e95edf0edae71 was the correct commit

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
rebased with master due to branch conflicts

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
ci:test:relevant

**Comment by marcogalluzzi** [comment]
ci:forward

**Pull request closed without merging**